lib/publiccloud/instance.pm

```diff
@@ -165,9 +172,16 @@ sub ssh_script_run {
     my $self = shift;
     my %args = testapi::compat_args({cmd => undef}, ['cmd'], @_);
     my $ssh_cmd = $self->_prepare_ssh_cmd(%args);
+    $args{timeout} //= SSH_TIMEOUT;
+    if (exists $args{continue_after_timeout} && $args{continue_after_timeout}) {
+        my $external_timeout = $args{timeout};
+        $args{timeout} = $args{timeout} + 20;
+        $ssh_cmd = "timeout --foreground -k 10s $external_timeout " . $ssh_cmd;
```

Comment:
There must be justification in the form of comments on why these values were chosen.

Reply:
done